index.js

```diff
@@ -759,3 +759,20 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.oneOf = function (value) {
+  for (var _len = arguments.length, types = Array(_len > 1 ? _len - 1 : 0), _key = 1; _key < _len; _key++) {
```

Comment:
I'm not opposed to adding the method, especially since `Array#some` requires ES5 or es5-shim. I'm confused about the use case though - when do you have a value that 1) could be multiple kinds of thing, and 2) it's OK that it's not a specific kind of thing?